tests/ui/traits/item-privacy.stderr

```diff
@@ -8,9 +8,9 @@ LL |     S.a();
    |       ^
    |
    = help: items from traits can only be used if the trait is implemented and in scope
-help: trait `A` which provides `a` is implemented but not in scope; perhaps you want to import it
+help: trait `A` which provides `a` is implemented but not in scope; perhaps you want to use the inaccessible trait
```

Comment:
"perhaps you want to use the inaccessible trait" seems inactionable.

Why don't we collapse this into a single help:

`help: trait method::A provides a which is implemented but not in scope`

so we're just noting it rather than trying to tell the user that they need to use a trait that is unusable